windnode_abw/model/region/model.py

```diff
@@ -647,8 +650,26 @@ def create_flexopts(region=None, datetime_index=None, nodes_in=[], scn_data={}):
                 # DSM (households) #
                 ####################
                 if scn_data['flexopt']['dsm']['enabled']['enabled'] == 1:
-                    # ToDo: Do not forget to replace the el. sinks above!
-                    # ToDo: (do not create the sinks in line 161 ff.)
-                    raise NotImplementedError
+                    # if DSM is enabled normal Sinks l.161 ff. will be deactivated
+
+                    nodes.append(
+                        solph.custom.SinkDSM(
+                            label='dsm_el_{ags_id}_b{bus_id}'.format(
+                                ags_id=str(mun.Index),
+                                bus_id=busdata.Index
+                            ),
+                            inputs={bus_in: solph.Flow()},
+                            demand=region.dsm_ts['Lastprofil'][mun.Index]/len(mun_buses),
```

Comment:
I thought a builtin type is expected by oemof, but you pass a pandas Series obj as iterable. Is this valid? Besides, the parameter list of your SinkDSM component says array.